win32/streamlinkrc

```diff
@@ -1,66 +1,87 @@
-# Format is option=value. Lines starting with a # is considered comments
-# and are ignored.
-
-# By default streamlink will attempt to locate VLC on your system
-# and use that, but you can also specify the location of a player
-# yourself.
-
-# Important: You must use a quoted path if there are spaces in the path. This
-# is because the player command is parsed like a shell command to allow
-# parameters to be passed to the player.
-
-# Here is a few examples of players:
-
-# VLC
-#player="C:\Program Files (x86)\VideoLAN\VLC\vlc.exe"
-#player="C:\Program Files\VideoLAN\VLC\vlc.exe"
-# Using --file-caching is recommended, but is only supported in VLC 2.0+
-#player="C:\Program Files (x86)\VideoLAN\VLC\vlc.exe" --file-caching=5000
-#player="C:\Program Files\VideoLAN\VLC\vlc.exe" --file-caching=5000
-
-# MPC-HC, must be at least version 1.7 to be used
-#player="C:\Program Files (x86)\MPC-HC\mpc-hc.exe"
-#player="C:\Program Files\MPC-HC\mpc-hc64.exe"
-
-
-# Use this if you want to transport the stream to the player via a named pipe.
-#player-fifo
-
-# Use this if you want to transport the stream to the player via HTTP.
-#player-http
-
-# Use this if you want streamlink to only pass a URL to your player and
-# let it handle the transport of the stream itself.
-#player-passthrough=http,hls,rtmp
-
-# By default streamlink will close the player when stream is over.
-# Use this option to let the player stay or close itself instead.
-#player-no-close
-
-# Use this option if you want streamlink to keep trying to access
-# the stream even if it goes offline or disconnects. Your player must
-# support HTTP and its playlist should be set to repeat mode.
-#player-continuous-http
-
-# Show console output from the video player
-#verbose-player
-
-# RTMP streams are downloaded using rtmpdump. Full path or relative path
-# to the rtmpdump exe should be specified here.
-#rtmpdump=C:\Program Files (x86)\Streamlink\rtmpdump\rtmpdump.exe
-rtmpdump=rtmpdump.exe
-
-# FFMPEG is used to mux separate video and audio streams in to a single
-# stream so that they can be played. The full or relative path to ffmpeg
-# or avconv should be specified here.
-#ffmpeg-ffmpeg=C:\Program Files (x86)\Streamlink\ffmpeg\ffmpeg.exe
-ffmpeg-ffmpeg=ffmpeg.exe
-
-# Log level, default is info
-#loglevel=debug
-
-# Number of threads to use when streaming HLS streams
-#hls-segment-threads=1
-
-# Number of threads to use when streaming HDS streams
-#hds-segment-threads=1
+# The format is option=value
+# Lines starting with a # are considered comments and are ignored.
+
+# Please see the Streamlink CLI documentation for all available options
+# and their descriptions. All leading dashes need to be ignored.
```

Comment:
I think an example here would be good such as "(e.g. `--default-stream STREAM` becomes `default-stream=STREAM`)".